client/jetpack-connect/woo-loader.tsx

```diff
@@ -8,7 +8,7 @@ import './woo-loader.scss';
 
 export type Stage = {
 	title: string;
-	image?: string | JSX.Element;
+	image?: JSX.Element;
```

Comment:
We should use `image?: React.ComponentType` here. The image is a component that returns JSX.element.

```suggestion
	image?: React.ComponentType
```